executor/aggregate.go

```diff
@@ -214,6 +217,7 @@ func (e *HashAggExec) Close() error {
 		e.childResult = nil
 		e.groupSet = nil
 		e.partialResultMap = nil
+		e.firstRowProcessed = 0
```

Comment:
I think move the line to Open() is better..  I'm so sorry for the previous useless comment.